lib/wallet/rpc.js

```diff
@@ -1394,10 +1387,11 @@ class RPC extends RPCBase {
       const hash = addr.getHash();
 
       if (value == null)
-        throw new RPCError(errs.INVALID_PARAMETER, 'Invalid parameter.');
+        throw new RPCError(errs.INVALID_PARAMETER, 'Invalid amount.');
 
       if (uniq.has(hash))
```

Comment:
Pretty sure this error will never happen, because you must pass this function an object of shape `{address: amount}` and there is no way to have the same key twice in an object...

Reply:
That sounds correct, yeah.

Reply:
I think it likely came from a port from `bitcoind` as it has the same check, perhaps the implementation of json parsing with `univalue` can have duplicate keys.